tool/gravity/cli/register.go

```diff
@@ -139,11 +139,16 @@ func RegisterCommands(app *kingpin.Application) *Application {
 	g.ResumeCmd.Force = g.ResumeCmd.Flag("force", "Force execution of specified phase.").Bool()
 	g.ResumeCmd.PhaseTimeout = g.ResumeCmd.Flag("timeout", "Phase execution timeout.").Default(defaults.PhaseTimeout).Hidden().Duration()
 
-	g.ReconfigureCmd.CmdClause = g.Command("reconfigure", `Reconfigure the advertise address used by the node. Supported for single-node clusters only. Cluster services must be stopped (e.g. using "gravity system stop") prior to initiating the operation.`)
-	g.ReconfigureCmd.Path = g.ReconfigureCmd.Arg("path", "Path to the directory with the unpacked cluster image. Defaults to the current directory.").String()
-	g.ReconfigureCmd.AdvertiseAddr = g.ReconfigureCmd.Flag("advertise-addr", "New advertise address the node will use. Must be present on the node. Will be auto-selected if not specified.").String()
-	g.ReconfigureCmd.FromService = g.ReconfigureCmd.Flag("from-service", "Run in service mode.").Hidden().Bool()
-	g.ReconfigureCmd.Confirm = g.ReconfigureCmd.Flag("confirm", "Suppress confirmation prompt.").Bool()
+	// TODO(r0mant): Add flags --planet and --teleport to stop/start command.
+	g.StopCmd.CmdClause = g.Command("stop", "Stop Gravity services on the node.")
+	g.StopCmd.Confirmed = g.StopCmd.Flag("confirm", "Suppress confirmation prompt.").Bool()
+	g.StopCmd.Disable = g.StopCmd.Flag("disable", "Disable systemd services as well.").Bool()
+
+	g.StartCmd.CmdClause = g.Command("start", "Start Gravity services on the node.")
+	g.StartCmd.AdvertiseAddr = g.StartCmd.Flag("advertise-addr", "New advertise address the node will use. Must be present on the node.").String()
```

Comment:
Is this parameter validated?

Reply:
Yes, same way as the advertise-addr parameter for gravity install.